nautobot/utilities/utils.py

```diff
@@ -503,5 +517,19 @@ def get_table_for_model(model):
     return get_related_class_for_model(model, module_name="tables", object_suffix="Table")
 
 
+def get_content_type_string(model):
+    """Return the the content type string given model.
+
+    Returns:
+        The name of the model in "{app_label}.{model_name}" format
+    """
+    if not inspect.isclass(model):
+        model = model.__class__
+    if not issubclass(model, Model):
+        raise TypeError(f"{model} is not a subclass of Django Model class")
+
+    return f"{model._meta.app_label}.{model._meta.model_name}"
```

Comment:
TODO: Review change to: https://github.com/nautobot/nautobot/blob/ac1768ab78448955c42425144ddfa91509a13aa1/nautobot/extras/plugins/validators.py#L23